### ISSUE TITLE
An Awesome Eleventy List 🥳

### ISSUE THREAD

**Issue by chrissy-dev** [11ty/11ty-community#64]
This isn't an issue - more of a discussion. 

I personally find [Awesome Lists](https://github.com/sindresorhus/awesome) a super useful resource.

I had a look for an Eleventy focused one but couldn't find it. I've made a brief start but would ideally be something the community maintained - throw PRs at it 🥳. 

I just have a couple of questions:
- If this is not something the community feel is required, it can quietly disappear. Is it? 
- If @zachleat (or anyone) would prefer if it was transferred to the 11ty organisation - let me know 😃.

The list is here: [Awesome Eleventy](https://github.com/chrisssycollins/awesome-eleventy).

**Comment by tonystar** [comment]
Exactly! Bob's `11tybundle.dev` success proves that a single centralized `11ty.dev` can't meet all the needs of the Community (and it does not have to!)

For example, https://www.11ty.dev/docs/starter/ doesn't actually help users choose a starter. While it lists everything and helps with discovery, it's more useful for search bots and LLMs than for real people, looking for insights from other real people behind those starters...

So:

> I don't see many added value here, adding those contributions to the website looks more useful.

I'd respectfully counter-argue this, and support original @chrissy-dev 's idea. An Awesome list helps navigate various community-driven resources. For starters, it can aggregate links to https://www.11ty.dev/docs/starter/, https://11tybundle.dev/starters/, https://buildawesome.one/starters/, and other maintained collections.

In short, I view https://www.11ty.dev/ as a centralized, highly moderated "11ty Wikipedia," whereas an Awesome 11ty list serves as a more open, decentralized, and fast-evolving "11ty Digital Garden."
